lib/helium/sensor.rb

```diff
@@ -59,8 +53,7 @@ def as_json
         name: name,
         mac: mac,
         ports: ports,
-        last_seen: last_seen,
-        labels: labels
```

Comment:
the only thing that used it was my hackish `sensor/:id/label` I put in our route.  So I will change when this drops.